### PR TITLE
Release notes cleanup

### PR DESCRIPTION
**Pull request by TTitcombe**
**Description of work.**
This PR cleans-up the release notes for v4.0.
Spelling, grammar, and ensuring a uniform style throughout the release notes.
Project recovery release note screenshots have been updated.

`Muon` and `SANS` release notes are being addressed in separate PRs.

**To test:**
Read the changes
Doc test passes

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by martyngigg — DISMISSED** [review]
Looks good :+1: 

@mantidproject/gatekeepers Could we get this merged soon as it will start attracting merge conflicts.

File: docs/source/release/v4.0.0/diffraction.rst [hunk 1]
```diff
@@ -62,57 +57,54 @@ Bugfixes
...
 - Focus now checks if the vanadium for a run is already loaded before loading it in to prevent reloading the same vanadium multiple times.
 - :ref:`SaveReflections <algm-SaveReflections>` now supports saving indexed modulated peaks in the Jana format.
-- `PyStoG <https://pystog.readthedocs.io/en/latest/>`_ has been added as an external project
+- `PyStoG <https://pystog.readthedocs.io/en/latest/>`_ has been added as an external project.
 
 Bugfixes
 ########
 
-- multiple_scattering flag is now optional for Polaris focus when absorb_correction is true.
+- Multiple_scattering flag is now optional for Polaris focus when absorb_correction is true.
```
Comment (DanNixon) [inline docs/source/release/v4.0.0/diffraction.rst]: `multiple_scattering` may be better as unformatted text (and with correct captialisation) if it is something in code (which this suggests it is).

**Review by soininen — CHANGES_REQUESTED** [review]
Broken links in framework.rst: 
SetGoniometer
AlignAndFocusPowderFromFiles
AlignAndFocusPowder (multiple)
DiffractionFocussing

FileFinder.findRuns has troubles in formatting

**Review by DanNixon — CHANGES_REQUESTED** [review]
One small suggestion.

**Review by soininen — APPROVED** [review]

**Review by DanNixon — APPROVED** [review]

**Pull request merged**